google_takeout_parser/models.py

```diff
@@ -105,13 +105,27 @@ def key(self) -> int:
 class Location(BaseEvent):
     lng: float
     lat: float
+    address: Optional[str]
```

Comment:
Doesnt look like `Location History/Location History.json` has this data, instead of adding it to the Location here, I think for `cachew` purposes it may be nicer to just add all this info onto the place visit directly.

Reusing models simplicity's sake may be nice in most cases but here it makes things more complicated since it interferes with other parsers/and is being serialized to the database.

So, something like:

```python
@dataclass
class PlaceVisit(BaseEvent):
    lng: float
    lat: float
    address: Optional[str]
    accuracy: Optional[int]
    placeId: Optional[str]
    # then the other placevisit fields
    startTime: datetime
    endTime: datetime
    visitConfidence: Optional[int]
```